spec/factories/competitions.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :competition do
+    code_name 'Code name'
+    start_date '2015-09-19'
+    end_date '2015-09-20'
+  end
```

Comment:
We should add a `with_projects` trait to optionally create projects with the competition.